docs/schema-update-guide.md

```diff
@@ -2,24 +2,31 @@
 
 All schemas are located at [store/migration](https://github.com/bytebase/bytebase/tree/main/store/migration). Bytebase schema release follows semantic versioning. This document goes over the scheme update process during development and release.
 
+We use different schemas for [dev](https://github.com/bytebase/bytebase/tree/main/store/migration/dev) and [release](https://github.com/bytebase/bytebase/tree/main/store/migration/release). The `LATEST.sql` and `LATEST_DATA.sql` will be the schema and data to initialize a new database. The version directory such as `1.1` contains all the DDL statements used to migrate from previous version of schema such as `1.0`. There should only one version in dev schema directory.
+
 ## Development
-We use different schema versions for [dev](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_dev.go) and [release](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_release.go). 
 
-If schema versions in dev/release profiles are the same (e.g. 1.1.0), which means we are starting a new MINOR version update cycle.
-1. Create a new directory under [migration directory](https://github.com/bytebase/bytebase/tree/main/store/migration) with the new version, e.g. 1.2.0.
-2. Copy over the schema and data file from the previous version. e.g. [1.1.0/latest.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest.sql) and [1.1.0/latest_data.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/latest_data.sql). And make a PR for the changes above.
-3. Bump up the MINOR version in the [dev schema version](https://github.com/bytebase/bytebase/blob/main/bin/server/cmd/profile_dev.go), e.g. 1.1.0 will be 1.2.0.
-4. Add a new SQL file with DDL statements with a descriptive name, e.g. [0000_github_com.sql](https://github.com/bytebase/bytebase/blob/main/store/migration/1.1.0/0000_github_com.sql).
-5. Update latest.sql and latest_data.sql data file w.r.t. DDL changes.
-6. Update the demo data in the test directory.
+Let's take an example that our release is at version `1.5` and dev at version `1.6`.
+
+If there is `1.6/0000__placeholder.sql` file in the dev version directory,
```

Comment:
Can we simplify this without adding 0000__placeholder in advance and just create an actual migration file on demand?

Just offer user a single option instead of multiple ones

Reply:
Dev and release are using the same schema then and would require some coding change (it's already quite complex). Having 0000__placeholder.sql may be a good thing because people can just follow the naming convention. Or maybe we can just keep this file all the time around?

Reply:
Let's see how it actually works.